README.md

```diff
@@ -1,17 +1,17 @@
 Numix icon theme
 ================
-Numix is the official icon theme from the [Numix Project](http://numixproject.org). It is heavily inspired by, and based upon parts of the Elementary, Humanity and Gnome icon themes. Designed to be used along side the application icon theme [Numix Circle](https://github.com/numixproject/numix-icon-theme-circle). This readme provides information on [installation](https://github.com/numixproject/numix-icon-theme#installation) and [icon requests](https://github.com/numixproject/numix-icon-theme#icon-requests). Licensed under the GPL-3.0+
+Numix is the official icon theme from the [Numix Project](http://numixproject.org). It is heavily inspired by, and based upon parts of the Elementary, Humanity and Gnome icon themes. Designed to be used along side the application icon theme [Numix Circle](https://github.com/numixproject/numix-icon-theme-circle). This readme provides information on [installation](https://github.com/numixproject/numix-icon-theme#install-it) and [icon requests](https://github.com/numixproject/numix-icon-theme#icon-requests). Licensed under the GPL-3.0+
+
+## Install it
 
-### Installation
 If using this with one of our app icon themes (like [Numix Circle](https://github.com/numixproject/numix-icon-theme-circle)) make sure you install both parts using the same method. This makes sure that the panel icons keep working as intended.
 
-If you use Ubuntu or any of its derivatives (including Mint and elementary OS) you can use our Numix PPA. Fire up a Terminal and run the following:
+### Official Distro Packages
+|Distro|Install Command/Links|
+|:----:|:----:|
+|[![debian](https://www.debian.org/logos/openlogo-nd-25.jpg)](https://www.debian.org/)&nbsp;![ubuntu](https://dl.dropboxusercontent.com/u/60521097/ubuntu_orange_hex.png)|`apt install numix-icon-theme`|
 
-```bash
-sudo add-apt-repository ppa:numix/ppa
-sudo apt-get update
-sudo apt-get install numix-icon-theme
-```
+### Unofficial Distro Packages
```

Comment:
If/when Travis CI is setup, it could be used to do automated releases once in a while.

Like, say, once per month or every X new icons, a new tag is set and Travis CI is setup to deploy on tagged commits. It'll run tests (SVG cleaning, etc), compile everything and then deploy to Github Releases, which stable distros can then use to make packages (debian/watch works with Github Releases and Tags)